pkg/autohealing/config/config.go

```diff
@@ -83,12 +86,13 @@ type kubeConfig struct {
 // NewConfig defines the default values for Config
 func NewConfig() Config {
 	return Config{
-		DryRun:               false,
-		CloudProvider:        "openstack",
-		MonitorInterval:      30 * time.Second,
-		MasterMonitorEnabled: true,
-		WorkerMonitorEnabled: true,
-		LeaderElect:          true,
-		CheckDelayAfterAdd:   10 * time.Minute,
+		DryRun:                  false,
+		CloudProvider:           "openstack",
+		MonitorInterval:         60 * time.Second,
```

Comment:
Is there reason why we change this to 60?

Reply:
Good question. The default report interview from kubelet to kube-apiserver is 40s, so it's possible that the node status hasn't been updated within 30s. And based on our experience on prod, 30s is a bit too often. So I'm proposing to change it to a number just more than 40s so 60s is a good balance. I will update the release to cover this.

Reply:
I'm fine with 60 secs, but I wonder is there any proper way to notify users about the default config change. releasenote?:)

Reply:
I have already updated the relesse note.